internal/cephfs/volume.go

```diff
@@ -134,6 +140,44 @@ func createVolume(ctx context.Context, volOptions *volumeOptions, cr *util.Crede
 	return nil
 }
 
+func resizeVolume(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, volID volumeID, bytesQuota int64) error {
```

Comment:
Maybe add a comment about the fallback logic that happens in this function? The `ceph fs subvolume resize` command was added in recent Ceph versions, in case it is not available apply the resize through the ol `ceph volume create` command?